Source/options.cpp

```diff
@@ -1473,7 +1473,7 @@ void KeymapperOptions::KeyReleased(uint32_t key) const
 
 	// Check that the action can be triggered and that the chat textbox is not
 	// open.
-	if (!action.actionReleased || (action.enable && !action.enable()) || talkflag)
+	if (!action.actionReleased || (action.enable && !action.enable()) || ((talkflag || dropGoldFlag) && action.key != "Screenshot"))

```

Comment:
is p hardcoded to something somewhere?

Reply:
No but it could cause some issues when trying to write a chat message "ick u the otions before using the ortal"

Reply:
😂 good point. I guess it can be checked against characters that can be typed first to decided whether or not print screen will work. 

But for what it's worth, I did check `p` and it didn't seem to work in main menu while F9 was working. I can look into this more tomorrow.

Reply:
thanks

Reply:
I think this is resolved now, feel free to take a look when you get a chance.